Gemfile

```diff
@@ -1,59 +1,59 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.7.1'
-gem 'activerecord-session_store', '1.0.0'
-gem 'unicorn', '4.6.2'
-gem 'pg', '0.18.4'
-gem 'optic14n', '2.0.1'     # Ideally version should be synced with bouncer
-gem 'gds-sso', '12.0.0'
-gem 'govuk_admin_template', '4.2.0'
-gem 'plek', '1.12.0'
-gem 'htmlentities', '4.3.4'
-gem 'kaminari', '0.16.3'
+gem 'rails', '5.1.1'
+gem 'activerecord-session_store'
+gem 'unicorn'
+gem 'pg'
+gem 'optic14n' # Ideally version should be synced with bouncer
+gem 'gds-sso'
+gem 'govuk_admin_template'
+gem 'plek'
+gem 'htmlentities'
+gem 'kaminari'
 gem 'paper_trail', '4.1.0'
-gem 'google-api-client', '0.8.6'
-gem 'gds-api-adapters', '36.0.1'
-gem 'mlanett-redis-lock', '0.2.7'
-gem 'whenever', '0.9.4'
-gem 'gretel', '3.0.8'
-gem 'acts-as-taggable-on', '3.5.0'
+gem 'google-api-client'
+gem 'gds-api-adapters'
+gem 'mlanett-redis-lock'
+gem 'whenever'
+gem 'gretel'
+gem 'acts-as-taggable-on'
 gem 'select2-rails', '3.5.7'
-gem 'activerecord-import', '0.12.0'
-gem 'sidekiq', '4.1.1'
-gem 'redis-namespace', '1.5.2'
+gem 'activerecord-import'
+gem 'sidekiq'
+gem 'redis-namespace'
 
 # We use Errbit for tracking exceptions, which needs the airbrake gem. Config
 # for Errbit is in alphagov-deployment.
-gem 'airbrake', '~> 4.3.0'
```

Comment:
Airbrake 5 breaks deploy notifications in our errbit.  We've a branch of 4.3 that fixes dep warnings in rails 5 though. 
 See https://docs.publishing.service.gov.uk/manual/upgrade-rails.html#airbrake for more details.